win/stdatomic.c

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+#include <intrin.h>
+
+#include "stdatomic.h"
```

Comment:
Not sure if #include "stdatomic.h"  has the same behavior as  #include <stdatomic.h> on win. In other c files, the code is #include <stdatomic.h>.

Reply:
In visual studio include "" means to search header in the project.
include <> means to seach the header in C/C++ libraries first.
Since I don't have stdatomic in the stsndart libraries, VS tries to find this file in the project.